spec/facades/food_facade_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe FoodFacade do
+  it 'formats all desired info' do
+    results = FoodFacade.foods_by_ingredient('sweet potatoes')
+
+    expect(results.total_results).to eq(44128)
+    expect(results.query).to eq('Sweet potatoes')
+    expect(results.foods.count).to eq(10)
```

Comment:
Solid. I might also check that `results.foods` is an `Array` and the first object in the array is a `Food` object. Also, would be good to stub out the Service API call that this facade method depends on .